pkgs/development/python-modules/lb-matching-tools/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  regex,
+  pytestCheckHook,
+  nix-update-script,
+}:
+
+buildPythonPackage rec {
+  pname = "lb-matching-tools";
+  version = "2024.01.30.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "metabrainz";
+    repo = "listenbrainz-matching-tools";
+    tag = "v${version}";
+    hash = "sha256-RQ4X6DKigQsNxaAWXB1meATKP+ddMUgkoAIyX8iIisU=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ regex ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "lb_matching_tools" ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
By default, [python libraries already have an update script](https://github.com/NixOS/nixpkgs/blob/7d19100fc0608a3a2e0109ebd36a75328eaef20e/pkgs/development/interpreters/python/mk-python-derivation.nix#L423), so this is actually not required.

Same suggestion for other derivations that don't need to be updated from branch.

Reply:
I like being explicit. This shows at a glance that it has an update script and it's supposed to work rather than it just happens to work.

Reply:
I indeed have imagined that and hope it to be the future. At that point, I would prefer only putting something when a change from default needs to be made.

> I also think it would be nice if we started doing this for more builders

It is to some degree. I can't remember every time I've run into it, but for example, vscode extensions.